ratelimiter/src/main/java/com/flipkart/varadhi/qos/RateLimiter.java

```diff
@@ -0,0 +1,7 @@
+package com.flipkart.varadhi.qos;
+
+public interface RateLimiter<R, L> {
```

Comment:
reverse the order of type params.

<params, result>

Reply:
Got it, Kept it descriptive rather than L, R